client/src/featureform/register.py

```diff
@@ -161,7 +161,7 @@ def register_table(
         return self.__registrar.register_primary_data(
             name=name,
             variant=variant,
-            location=SQLTable(table),
+            location=SQLTable('"{}"'.format(table)),
```

Comment:
We were getting errors initially because snowflake needs " " to be part of the table name string, but the code didnt do that. So snowflake wouldnt recognize the table name.

Reply:
this change break logic when register table bigquery